app/src/main/java/com/babylon/wallet/android/presentation/settings/authorizeddapps/dappdetail/UnknownDAppComponentsSheetContent.kt

```diff
@@ -0,0 +1,120 @@
+import com.babylon.wallet.android.presentation.ui.composables.BackIconType
+import com.babylon.wallet.android.presentation.ui.composables.RadixCenteredTopAppBar
+import com.babylon.wallet.android.presentation.ui.composables.Thumbnail
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+
+@Composable
+fun UnknownDAppComponentsSheetContent(
+    onBackClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    unknownDAppComponents: ImmutableList<String>
+) {
+    Column(modifier = modifier) {
+        val title = stringResource(id = R.string.transactionReview_unknownComponents, unknownDAppComponents.size)
+        RadixCenteredTopAppBar(
+            title = title,
+            onBackClick = onBackClick,
+            contentColor = RadixTheme.colors.gray1,
+            backIconType = BackIconType.Close
+        )
+        HorizontalDivider(color = RadixTheme.colors.gray5)
+        val lastItem = unknownDAppComponents.last()
+        LazyColumn(
+            contentPadding = PaddingValues(vertical = RadixTheme.dimensions.paddingDefault),
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(RadixTheme.dimensions.paddingDefault)
+        ) {
+            items(unknownDAppComponents) { unknownComponentAddress ->
+                Column {
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(RadixTheme.dimensions.paddingSmall)
+                            .padding(vertical = RadixTheme.dimensions.paddingDefault),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Thumbnail.DApp(
+                            modifier = Modifier.size(44.dp),
+                            dapp = null,
+                            shape = RadixTheme.shapes.roundedRectXSmall
+                        )
+                        Spacer(modifier = Modifier.width(RadixTheme.dimensions.paddingDefault))
+                        Column {
+                            Text(
+                                text = stringResource(id = R.string.common_component),
+                                style = RadixTheme.typography.body1Header,
+                                color = RadixTheme.colors.gray1,
+                                overflow = TextOverflow.Ellipsis
+                            )
+
+                            ActionableAddressView(
+                                address = unknownComponentAddress,
+                                textStyle = RadixTheme.typography.body2Regular,
+                                textColor = RadixTheme.colors.gray1
+                            )
+                        }
+                        Spacer(modifier = Modifier.weight(1f))
+                        Icon(
```

Comment:
This doesn't do anything for now, need to confirm it and also if we want to keep it or not. Once confirmed will either remove it or modify and convert draft to PR